server/config/router.js

```diff
@@ -76,47 +39,52 @@ module.exports = function (nextjs, koa) {
   // Legacy Wordpress permalinks
   // e.g. /2017/09/07/universal-service-a-fuel-rats-thargoid-cartoon
   router.get('/:year/:month/:day/:slug', async (ctx, next) => {
-    let {
+    const {
       day,
       month,
       slug,
       year,
     } = ctx.params
 
-    let dayIsValid = parseInt(day) && (day.length === 2)
-    let monthIsValid = parseInt(month) && (month.length === 2)
-    let yearIsValid = parseInt(year) && (year.length === 4)
+    const dayIsValid = parseInt(day, 10) && (day.length === 2)
```

Comment:
Why are you using safeParseInt in some places but not here?

Reply:
cuz thats a server end thing to help resolve old permalinks. I'm not too worried about it as I don't generally expect user-generated input to pass through it. that and this function scares me

Reply:
sounds good to me